rikaicontent.js

```diff
@@ -321,19 +321,23 @@ var rcxContent = {
 	keysDown: [],
 	lastPos: { x: null, y: null},
 	lastTarget: null,
+	altGraphKey: false,
 
 	onKeyDown: function(ev) { rcxContent._onKeyDown(ev) },
 	_onKeyDown: function(ev) {
 		//console.log("key=" + ev.key + " keyCode=" + ev.keyCode + ' charCode=' + ev.charCode + ' detail=' + ev.detail);
  
+		if (ev.key === "AltGraph") {

```

Comment:
Identical. This is chromium's stupidity. On russian layout it generates `Alt` event for left Alt and `AltGraph` for right Alt. Even 'Right Alt+Left' doesn't take you back in history on russian layout, while 'Left Alt+Left' does.